src/main/java/org/smartregister/pathevaluator/task/TaskHelper.java

```diff
@@ -3,20 +3,50 @@
  */
 package org.smartregister.pathevaluator.task;
 
+import org.joda.time.DateTime;
 import org.smartregister.domain.Action;
 
 import com.ibm.fhir.model.resource.Resource;
+import org.smartregister.domain.Task;
+import org.smartregister.pathevaluator.PathEvaluatorLibrary;
+import org.smartregister.pathevaluator.dao.TaskDao;
+
+import java.util.UUID;
 
 /**
  * @author Samuel Githengi created on 06/15/20
  */
 public class TaskHelper {
 
+	private TaskDao taskDao;
```

Comment:
get this from `taskProvider` on `PathEvaluatorLibrary.getInstance();`

Reply:
Done